packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.html

```diff
@@ -1,32 +1,54 @@
 <template>
-  <li class="slds-grid">
-    <div onclick={onSelect} class="facet__value-container facet__value-option slds-grid slds-grid_vertical-align-center slds-size_1-of-1 slds-p-around_x-small">
-      <template if:false={displayAsLink}>
-        <lightning-input 
-          type="checkbox"
-          class="facet__value-checkbox"
-          variant="label-hidden"
-          label="Facet value option"
-          name="facet value checkbox"
-          onchange={onSelect}
-          checked={isChecked}
-        ></lightning-input>
+  <div
+    role={divRole}
+    aria-checked={isChecked}
+    onclick={onSelect}
+    class="facet__value-container facet__value-option slds-grid slds-grid_vertical-align-center slds-size_1-of-1 slds-p-around_x-small"
+    tabindex="0"
+    onkeydown={onKeyDown}
+    aria-label={ariaLabelVoiceOver}
+  >
+    <template if:false={displayAsLink}>
+      <lightning-input
+        tabindex="-1"
+        type="checkbox"
+        class="facet__value-checkbox"
+        variant="label-hidden"
+        label="Facet value option"
+        name="facet value checkbox"
+        onchange={onSelect}
+        checked={isChecked}
+      ></lightning-input>
+    </template>
+    <template if:true={isChecked}>
+      <span
+        class="slds-truncate slds-var-m-right_xx-small slds-var-m-top_xxx-small nowrap facet__value-text facet__value_selected"
+        >{formattedFacetValue}</span
+      >
```

Comment:
You didn't change anything there right? Is it a formatter that changed the look of this file?

Reply:
For the HTML formatter, it seems in .prettierignore file, all html files under lwc folder is excluded to use prettier as formatter. Which formatter should be applied to html files under dir lwc?